src/tests/edit.test.js

```diff
@@ -0,0 +1,39 @@
+import edit from '../edit.js';
+import { saveLocal } from '../live.js';
+
+jest.mock('../live.js', () => ({
```

Comment:
- [ ] You have done well adding tests for editing descriptions and removing all completed tasks, but you are missing tests for updating the `completed` status of a task. Kindly add that.